docs/recipes/svelte.md

```diff
@@ -0,0 +1,68 @@
+# Usage with Svelte
+
+XState integrates great with Svelte, and especially [Svelte stores!](https://svelte.dev/docs#svelte_store)
+
+The only gotcha is that you'll need this extra package.
+```js
+npm install @rollup/plugin-replace --save-dev
+```
+Then add this to the plugins array in rollup.config.js.
+```js
+replace({ 
+    'process.env.NODE_ENV': process.env.NODE_ENV
```

Comment:
just an additional note: this would only work if `NODE_ENV` is set for your env, is it a common practice for it to be set when working with Svelte? an alternative for this would be:
```js
replace({ 
  'process.env.NODE_ENV': JSON.stringify('production') // or JSON.stringify('development')
})
```